test/Image2.rb

```diff
@@ -1066,12 +1066,12 @@ def test_levelize_channel
   #       end
   #
   #       res = nil
-  #       assert_nothing_raised do
+  #       expect do
```

Comment:
And here. There are so some more in the changes but not sure if you want to change this so I will stop here for now.